.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -17,7 +17,7 @@ body:
     id: reproduction
     attributes:
       label: Reproduction
-      description: A link to a repository that reproduces the issue. Explaining how to reproduce is generally not enough. It pushes the burden of creating a reproduction project onto a small set of volunteer maintainers and isn't scalable. If no reproduction is provided within a reasonable time-frame, the issue will be closed.
+      description: A link to a repository that reproduces the issue. Explaining how to reproduce is generally not enough. It pushes the burden of creating a reproduction project onto a small set of volunteer maintainers and isn't scalable. If no reproduction is provided, the issue will be closed.
```

how about adding this? it makes it easier to create repros, easier for us to verify them, and reduces repo clutter (both for us and for people opening issues)

```suggestion
      description: A link to a repository, or a fork of https://node.new/sveltekit, that reproduces the issue. Explaining how to reproduce is generally not enough. It pushes the burden of creating a reproduction project onto a small set of volunteer maintainers and isn't scalable. If no reproduction is provided, the issue will be closed.
```